tests/test_parsing.js

```diff
@@ -192,18 +193,33 @@ describe('Parser', function() {
 
     const expectedNode = createMemberTypeNode(
       createTypeNameNode('owner'),
-      'Mem\\ber');
+      'Mem\\ber',
+      'double');
 
     expect(node).to.deep.equal(expectedNode);
   });
 
   it('should return a member type node when \'owner."Mem.ber\\"\' arrived', function() {
-    const typeExprStr = 'owner."Mem.ber\\"';
+    const typeExprStr = 'owner."Mem.ber\\""';
```

Comment:
This shouldn't have been allowed, so changing the test to where a backslash at (or rather, toward) the end is allowed, before a quote.